cmd/create/cluster/cmd.go

```diff
@@ -1198,21 +1199,23 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	httpTokens := args.ec2MetadataHttpTokens
+	if httpTokens == "" {
```

Comment:
Does this change is needed for OCM-4646?

Reply:
Not directly, we could add something to UX improvements I suppose. The current prompt is a string which allows to enter anything and then we validate. I just don't think it makes sense when we have clear options for it

Reply:
Separated into a different PR https://github.com/openshift/rosa/pull/1664